tests/unit/drop-zone.spec.js

```diff
@@ -64,23 +64,32 @@ describe('DropZone', () => {
       { args: [], name: 'entered' },
     ]);
 
-    w.vm.client.addFile({ name: 'file1', data: '' });
+    const now = Date.now();
```

Comment:
Use a before hook to reset the date and provie a helper function to freeze the time.

Reply:
If it's only used here I had the impression an extra hook and helper function would be overkill.

Reply:
I hope the repo will grow so we will need a time freezer anyway. I think it's a good investment now.

Reply:
In the end used the mockdate library.